packages/databricks-vscode/src/extension.ts

```diff
@@ -585,6 +588,15 @@ export async function activate(
             );
         });
 
+    context.subscriptions.push(
+        commands.registerCommand("databricks.internal.clearOverrides", () => {
+            stateStorage.set("databricks.bundle.overrides", undefined);
+        })
+    );
+
```

Comment:
This is intended for resetting all the overrides for debugging. I intend to make it into a button in near future so users can reset all overrides.